pkg/api/schema.graphql

```diff
@@ -0,0 +1,83 @@
+schema {
+    query: Query
+}
+
+# Leave date as a string (specify YYYY-MM-DD) and convert later?
+# How do we do latest/earliest?
+#scalar Date
+
+# Query contains the entry points into the schema
+type Query {
+    #bug(id: Int!, datestamp: String = "latest"): Bug
+    # TODO: Add sorting argument for bugs? asc/desc + id/assigned_to/component/etc/etc...default to ???
```

Comment:
Is it worth implemented this?  Or just set a sensible default sorting that we would use on the frontend?  Users should be able to sort the objects to their own taste afterwards...?